docs/api/modules/Bundling-Modules.md

```diff
@@ -0,0 +1,186 @@
+[👈 Return to Overview](../API.md)
+
+# Bundling
+
+Within `one-app` you can bundle your modules once you've made changes to them. The bundled files are then loaded on the client and server.
+
+## Bundling Modules
```

Comment:
We may need to move this document as in hindsight to Recipes @nellyk as this definitely lends itself well to a guide.

Reply:
Agreed, i've moved this to recipes updated c5b191e